python/ray/ml/preprocessors/encoder.py

```diff
@@ -118,37 +118,40 @@ def __repr__(self):
 
 
 def _get_unique_value_indices(
-    dataset: Dataset, *columns: str
+    dataset: Dataset,
+    *columns: str,
+    drop_na_values: bool = False,
```

Comment:
Do we need to add this if it's not being used?

Reply:
It's actually being used in the Categorizer PR! Left it in here by mistake, but if we are going to merge that as well, it doesn't really make a huge difference to keep it in.